plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -119,6 +119,7 @@ def _onWriteStarted(self, output_device):
             }
             for container in global_container_stack.getContainers():
                 container_id = container.getId()
+                container_serialized = ""
```

Comment:
@Ghostkeeper: You are so right! For a reason I misunderstood the `continue` here.
Will remove that line in the new PR.